spring/src/main/java/io/seata/spring/annotation/datasource/EnableAutoDataSourceProxy.java

```diff
@@ -38,4 +38,10 @@
      * @return useJdkProxy
      */
     boolean useJdkProxy() default false;
+
+    /**
+     * Specifies which datasource bean are not eligible for auto-proxying
+     * @return
+     */
+    String[] exclude() default {};
```

Comment:
Do we need a include attribute?

Reply:
I don't think we need the include attribute, and exclude is only to avoid unnecessary proxy. In fact, even if the proxy, it won't have any impact.

Reply:
It makes sense. But I think `include` is an effective feature helping users to control auto proxying, avoiding unexpected situations, there is no harm to provide such a function. 
By the way, should `exclude` be `excludes`? Because the attribute is array type.

Reply:
done